docs/http.md

```diff
@@ -1,79 +1,239 @@
 # HTTP API
 
-When a Cog Docker image is run, it serves an HTTP API for making predictions. For more information, take a look at [the documentation for deploying models](deploy.md).
+> [!TIP]
+> For information about how to run the HTTP server, 
+> see [our documentation to deploying models](deploy.md).
+
+When you run a Docker image built by Cog, 
+it serves an HTTP API for making predictions.
```

Comment:
This would be a larger change (and therefore perhaps best made in a separate PR?) but we've been talking about moving away from using the term "prediction" in favor of more software-engineering-y terms like "run" (context: [style guide](https://www.notion.so/replicate/Style-guide-377cfdbd142540a9b3177734b63d0baf?pvs=4#8a0bfae63f514b2d955cf332cf429cfb)).

So this would become something like "it serves an HTTP API for running the model" ? Again, mostly just bringing it up here for discussion; this doc has so many mentions of prediction that I think if we do want to change the language we'd want to do that in a followup.

Reply:
Yes, please! Caveat:

You run models, models are run — I agree with that. When you run the model for inference, the method called is `predict` and the record / result of your running the model is a "prediction". I don't think we should start calling that a "run" (if only because of its scatalogical plural form, but also because that's a big API change).